manifests/t/TexasInstruments/TIConnectCE/6.0.1.344/TexasInstruments.TIConnectCE.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: TexasInstruments.TIConnectCE
+PackageVersion: 6.0.1.344

```

Comment:
The latest version on TI's website is `6.0.0.2688`, is this a beta version?

Reply:
I got it off here:
https://education.ti.com/en/software/details/en/CA9C74CAD02440A69FDC7189D7E1B6C2/swticonnectcesoftware#!

This is the latest version. I'll look into the driver software install as well, and change the link, may be easier to do in a new pr?